File: study/python_study/0724/1/2.py
__author__ = 'mingyu.zhang'
def findMinAndMax(L):
    Min=None
    Max=None
    if len(L) == 0:
        return (Min,Max)
    else:
        Min = L[0]
        Max = L[0]
        for l in L:
            if l < Min:
                Min = l
            if l > Max:
                Max = l
    return  (Min,Max)

if findMinAndMax([]) != (None, None):
    print('测试失败!')
elif findMinAndMax([7]) != (7, 7):
    print('测试失败!')
elif findMinAndMax([7, 1]) != (1, 7):
    print('测试失败!')
elif findMinAndMax([7, 1, 3, 9, 5]) != (1, 9):
    print('测试失败!')
else:
    print('测试成功!')

File: study/python_study/0724/1/2.js
/**
 * Created by mingyu.zhang on 2019/9/18.
 */
function nameinputcheckfunction() {
    var name = document.getElementById("nameinput").value
    try {
        if (name == null) throw "请输入";
        else if (typeof name != "string") throw "请输入正确格式";
        else if (name.length <= 0 || name.length >= 4) throw "长度不对";
        else {
            document.getElementById("nameinputchecknotice").innerHTML = "恭喜你";

        }
    }
    catch(err){
        document.getElementById("nameinputchecknotice").innerHTML=err
    }
}

var person={
    name:"Any",
    age:21,
    greet:function(){
        return this.name
    }
}

function people(name,age,gender){
    this.name=name,
    this.age=age,
     this.gener=gender,
      fullname=function(){
          return this.name+this.age
      }
}